src/util/matcher/stitcher.js

```diff
@@ -0,0 +1,104 @@
+        }
+      } else {
+        if (
+          processedPoints.points[i][j].lat > corners[2].lat &&
+          processedPoints.points[i][j].lat < center.lat
+        ) {
+          sectors.s11[i].push(processedPoints.points[i][j]);
+        } else {
+          sectors.s01[i].push(processedPoints.points[i][j]);
+        }
+      }
+    }
+  }
+  for (i=0; i<processedPoints.points.length; i++) {
+    sectors.population[i] = [];
+    sectors.population[i].push([
+      sectors.s00[i].length,
+      sectors.s01[i].length,
+      sectors.s10[i].length,
+      sectors.s11[i].length
+    ]);
+  }
+  var max = 0,
+    min = 10,
+    max_ = 0,
+    max_idx;
+  for (i in sectors.population[0][0]) {
+    if (sectors.population[0][0][i] > max) {
+      max = sectors.population[0][0][i];
+      max_idx = i;
+    }
+    if (sectors.population[0][0][i] < min) {
+      min = sectors.population[0][0][i];
+    }
+  }
+  if (max !== min) {
+    var coordinates = sectors[Object.keys(sectors)[max_idx]][0];
+    for (var u in processedPoints.points[0]) {
+      for (var v in coordinates) {
+        if (processedPoints.points[0][u] === coordinates[v]) {
+          if (processedPoints.confidences[0][u] > max_) {
+            max_ = processedPoints.confidences[0][u];
+          }
+        }
+      }
+    }
+    var best_point =
+      processedPoints.points[0][processedPoints.confidences[0].indexOf(max_)];
+    var corresponding_best_point =
+      processedPoints.points[1][processedPoints.confidences[0].indexOf(max_)];
+    document.querySelector(
+      "#map > div.leaflet-pane.leaflet-map-pane > div.leaflet-pane.leaflet-marker-pane"
+    ).innerHTML = "";
+    if(document.querySelectorAll("#map > div.leaflet-pane.leaflet-map-pane > div.leaflet-pane.leaflet-overlay-pane > svg > g > path").length) {
+      [].slice.call(document.querySelectorAll("#map > div.leaflet-pane.leaflet-map-pane > div.leaflet-pane.leaflet-overlay-pane > svg > g > path")).slice(0,3).map(function(x) {
+        x.parentNode.removeChild(x);
+      });
+    }
+    var lat_offset = -best_point.lat + corresponding_best_point.lat;
+    var lng_offset = -best_point.lng + corresponding_best_point.lng;
+    // revert this effect completely
+    overlay._corners[0] = [
+      processedPoints.images[1].getCorner(0).lat - lat_offset,
+      processedPoints.images[1].getCorner(0).lng - lng_offset
+    ];
+    overlay._corners[1] = [
+      processedPoints.images[1].getCorner(1).lat - lat_offset,
+      processedPoints.images[1].getCorner(1).lng - lng_offset
+    ];
+    overlay._corners[2] = [
+      processedPoints.images[1].getCorner(2).lat - lat_offset,
+      processedPoints.images[1].getCorner(2).lng - lng_offset
+    ];
+    overlay._corners[3] = [
+      processedPoints.images[1].getCorner(3).lat - lat_offset,
+      processedPoints.images[1].getCorner(3).lng - lng_offset
+    ];
+    var zoom_level = map.getZoom();
+    map.setView(overlay.getCenter(), (zoom_level%2?zoom_level+1:zoom_level-1));
```

Comment:
I was a bit disoriented when the map itself moved. Maybe it's not necessary?

Reply:
This actually is something weird. What's happening is that after updating the corners [here](https://github.com/publiclab/Leaflet.DistortableImage/blob/d4ce53adc505f2958e832c56b07a3d176060ecee/src/util/matcher/stitcher.js#L84-L99) the changes are actually reflected into the UI after I change the view, maybe there's some sort of cache refresh when map zooms out/in (since the map changes)? This would explain why it's happening. However, I can't yet find a documented way to trigger the same explicity, without the need of changing the view. What do you think?

Reply:
@rexagod haven't checked this for myself but I think you just need to call `overlay._reset()`?

Reply:
actually via #380 `setCorners` / `setCorner` methods I exposed should do the trick